docs/developer/checkout.mdx

```diff
@@ -327,3 +335,71 @@ A successful response would look like this:
   }
 }
 ```
+
+
+Here is the example of a `checkoutComplete` mutation where payment gateway requires additional data to finalize the payment:
+
+```graphql {2-4}
+mutation{
+  checkoutComplete(
+    checkoutId:"Q2hlY2tvdXQ6NWRiZmNmYjctMmFkYi00ZDU4LThmNTMtZjVmYjk0YjJmY2Ew",
+    paymentData:"{\"paymentMetodData\":\"some payment data\"}"){
```

Comment:
Can we provide a more meaningful example of payment data here? Something that looks like a real-life example? It would be easier to understand what is expected here.

Reply:
Hhm, not sure fo this. This field contains the data which will be passed to the payment plugin, payment plugin will use it to start processing the payment. It can be some advanced structure or some random string. I can add it but I am not sure if this will be helpful.